crimsobot/cogs/chat.py

```diff
@@ -16,13 +16,12 @@ def __init__(self, bot: CrimsoBOT) -> None:
     async def scatterbrain(self, ctx: commands.Context) -> None:
         """Short-term memory mania."""
 
-        channel = ctx.message.channel
-        messages = []
-
         # grab message contents (which are strings):
-        async for message in channel.history(limit=500):
-            if message.author.id != self.bot.user.id:
-                messages.append(message.content)
+        messages = [
+            message.content
+            async for message in ctx.channel.history(limit=500) 
+            if message.author.id != self.bot.user.id
+        ]
```

Comment:
Not a fan of this change. Is there a benefit to this that I'm not seeing? The list comprehension makes this less clear IMO.

Reply:
speed i believe, i personally find it more readable but it is up to taste